rsts/community/troubleshoot.rst

```diff
@@ -121,6 +121,17 @@ Troubles with handling large responses in ``FlyteRemote.sync``
 - To fix this, edit the ``flyte-admin-base-config`` config map using the command ``kubectl edit cm flyte-admin-base-config -n flyte`` to increase the ``maxMessageSizeBytes`` value.
 
 
+Issues with long running workflow executions (more than an hour)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- ```flytekit.exceptions.scopes.FlyteScopedUserException: An error occurred (ExpiredToken) when calling the AssumeRole operation: The security token included in the request is expired```
```

Comment:
Can you explain that in the case of long running workflow executions users might see this error being returned by flytekit?